packages/analyzer/src/features/analyse-phase/creators/createClassField.js

```diff
@@ -19,8 +19,16 @@ export function createField(node) {
   /**
    * Add TS type
    * @example class Foo { bar: string = ''; }
-   */ 
-  if(node.type) {
+   */
```

Comment:
Lets extract this into a separate handler (see `handlers.js`) that we can reuse in multiple places, something like `handleTypeInference(fieldTemplate, node)`:

```js
  switch(node?.initializer?.kind) {
    case ts.SyntaxKind.TrueKeyword:
    case ts.SyntaxKind.FalseKeyword:
      doc.type = { text: "boolean" }
      break;
    case ts.SyntaxKind.StringLiteral:
      doc.type = { text: "string" }
      break;
    case ts.SyntaxKind.NumericLiteral:
      doc.type = { text: "number" }
      break;
    case ts.SyntaxKind.NullKeyword:
      doc.type = { text: "null" }
      break;
    case ts.SyntaxKind.ArrayLiteralExpression:
      doc.type = { text: "array" }
      break;
    case ts.SyntaxKind.ObjectLiteralExpression:
      doc.type = { text: "object" }
      break;
  }
```

Reply:
PTAL